operations/restrictions.md

```diff
@@ -468,7 +468,106 @@ This reduces the overall number of restrictions and improves system performance.
 | `MinPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the minimum price per time unit. |
 | `MaxPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the maximum price per time unit. |
 
+## Clear restrictions
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change. Use of this operation must be enabled per enterprise. Please contact the Technical Partner Support team in order to enable it.
+
+Deletes restrictions that [match the conditions](#matching-conditions) using the [splicing algorithm](#splicing-algorithm).
+
+This operation is intended to be used in conjunction with [Set restrictions](#set-restrictions).
```

Comment:
```suggestion
This operation is intended to be used alongside [Set restrictions](#set-restrictions).
```
And we can delete the space beforehand, so this sentence follows immediately after the previous sentence.